src/unix/docker/dockerfiles/build/alpine/Dockerfile

```diff
@@ -38,4 +40,20 @@ RUN dotnet help
 
 # CLR Instrumentation Engine Build Prerequisites
 RUN apk update \
-    && apk add --no-cache libxml2-dev
\ No newline at end of file
+    && apk add --no-cache libxml2-dev
+
+# Install newer version of cmake. We need to run make commands inside the directory
+# that it is extracted to, so WORDIR is the easiest context to use.
+WORKDIR /usr/share/cmakeinst
+RUN wget -O cmake.tar.gz https://github.com/Kitware/CMake/releases/download/v3.20.2/cmake-3.20.2.tar.gz \
```

Comment:
Consider having some way of reading the cgmanifest file for the URL or version. That way we don't declare to CG that we are using one version and are actually using another version.

This is not something you have to do now, but is something to think about (if possible, since I know this is building within a Docker container).